lib/src/main/java/org/team100/lib/config/NotePoseDetector.java

```diff
@@ -107,7 +102,10 @@ public Double fieldRelativeY() {
      *         meters, rot value is bearing
      */
     public Translation2d RobotRelativeTranslation2d() {
-        return new Translation2d(getX(), getY());
+        if (m_notePosition24ArrayListener.getTranslation2d()[0].isPresent()) {
+        return m_notePosition24ArrayListener.getTranslation2d()[0].get();
+        }
+        return new Translation2d();
```

Comment:
if there's no value, return Optional.empty().  if you return a real value here, then you invite the caller to interpret it as a real value.